.travis.yml

```diff
@@ -7,9 +7,4 @@ services:
 
 script:
   - docker build "4" --tag "sonar-scanner-cli:4.1"
-  - echo "Docker image has been built!"
-notifications:
-  email: false
-  webhooks:
-    - secure: dc7zjwNNmFFqlt5uvngLlAV5OfUvTeesirBiIfjg44TsI+4DFZFA/1liBXgeuu5/oa218QOLtpKOVkZbgOL85z+H7F/22cUCyltd2h+3YlMK3vgzP35rY8F2gAyJJEeLnszOObS/JmhWn4hVpv1TVp/6vBtLMTjcV3Qfe/yKJ84/2se0o8jFF/8Q7qBISSzKxPCM1IsqEFlpfS8M8tGYNrkT/+ZN7dvvRaKGrhc997oMwqqFrx+vNRMST95fG/Bucsqs/6Ws9cvjNw8KYq4uLW1EE9IniNgfkHPDMZj2T9Tg6BKlGgVHv547dTdtAPQJhGsJPBQKx+Nt4YzTUCbBsu4itwNEursd0OHUgOim0t/h9ikITJYfMoQ4piLRQlOMH2e5Ih5Yo4K+IMzSqDfd2EhESafnXw/6dhLAsjVRstwFC9Aqn9jyTD2QZs/1ZPd4UdMR3HNeQK7MAd24IbQotwogX7ViSSij0+ynSynMouZAO7rOwUmDfHe2EN7WXHvhG0sT7Hm6OJQ779ruoWzAN1vayQA/1YYZq+jiGsVuLeIZuY/V9E73/lVxiD4kN9b+mOm93X2Mww6Lobic9G3C4l8j1Mb/UvqgkvYibORd1MdiYPNYgc0dIBTiv5AvZXuP214kD85+uNaEYekDdPd2ew0xsE8Ac77HERJ6D0EnadM=
-  on_start: always
+  - ./run-tests.sh "sonar-scanner-cli:4.1"
```

Comment:
Isn't the notification part the one which connects the build of the repo on Travis with Burgr?

Reply:
No, the webhook in other repos was used to trigger QA on cix which is not needed for this repo.